### DISCUSSION TITLE
Can't control on view height when soft keyboard appears because it covers typing message

### DISCUSSION THREAD

**Discussion by ibrahimahmed1998** [dotnet/maui#24151] [Q&A]
### Description

when I type message, the keyboard covers the typing message box and can't see or control input area because it not.NET MAUI Control it's related to WebView Component, how I can control in this view?

C# Code:

`MenuCurve.Source = new UrlWebViewSource { Url = url };
`

XML Code:

```
<WebView x:Name="MenuCurve" HorizontalOptions="CenterAndExpand" VerticalOptions="Start" BackgroundColor="White" ZIndex="50">

<WebView.Source>

<HtmlWebViewSource>

<HtmlWebViewSource.Html />

</HtmlWebViewSource>

</WebView.Source>

</WebView>
```

**Comment by stephenquan** [comment]
For controlling the soft keyboard input mode on Android see https://learn.microsoft.com/en-us/dotnet/maui/android/platform-specifics/soft-keyboard-input-mode?view=net-maui-8.0

**Reply by ibrahimahmed1998** [reply]
I used this doc and not effect on UI

**Reply by ibrahimahmed1998** [reply]
adding `using Microsoft.Maui.Controls.PlatformConfiguration.AndroidSpecific;
`then         
`App.Current.On<Microsoft.Maui.Controls.PlatformConfiguration.Android>().UseWindowSoftInputModeAdjust(WindowSoftInputModeAdjust.Resize);
` in my used page 
and same issue result

& when I add `Application.SetWindowSoftInputModeAdjust(this.Window, WindowSoftInputModeAdjust.Resize); 
`
the app crash

![image](https://github.com/user-attachments/assets/467a81ff-631c-4ccc-9a06-11dcc9bb8348)

**Reply by ibrahimahmed1998** [reply]
Hi @stephenquan 
err still exists

**Comment by ibrahimahmed1998** [comment]
!

**Comment by ibrahimahmed1998** [comment]
@stephenquan

**Comment by ibrahimahmed1998** [comment]
@stephenquan

**Comment by stephenquan** [comment]
@ibrahimahmed1998 no need to keep messaging me. I have given you my suggestion. We have used that documentation for ourselves and it works for us. I have no idea why it doesn't work for you. I cannot help you any further.
